front/app/modules/commercial/machine_translations/citizen/components/PostShowTranslatedBody.tsx

```diff
@@ -15,16 +17,18 @@ interface Props {
 
 const PostShowTranslatedBody = ({
   translateButtonClicked,
-  locale,
   postId,
   postType,
   body,
 }: Props) => {
+  const locale = useLocale();
+
```

Comment:
To confirm - machine translations are always used to translate the text into whatever locale the use is currently on right?

Reply:
Yes, that's correct :)